roles/config-rh-sso/tasks/configure-rh-sso.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: "Requesting Access Token"
+  uri:
+    url: "http://{{ external_ip }}:8080/auth/realms/master/protocol/openid-connect/token"
```

Comment:
Instead of `external_ip` I'd suggest making it more specific, such as `rh_sso_host`. Also, would be good to have a `rh_sso_port` that's defaulted to `8080` in  `defaults/main.yml`